src/steps/backup-restore/index.js

```diff
@@ -58,12 +58,12 @@ async function restoreBackup({ backupFile, databaseUrl, configuration }) {
   logger.info('Restore done');
 }
 
-async function createBackup(configuration) {
+async function createBackup(configuration, tablesToExcludeFromBackup) {
```

Comment:
C'est le moment d'aider le développeur qui va rajouter un argument après
```suggestion
async function createBackup({configuration, tablesToExcludeFromBackup}) {
```

Reply:
mis à jour